lib/src/Uncertainty/Model/UnionEvent.cxx

```diff
@@ -136,7 +136,7 @@ void UnionEvent::setEventCollection(const RandomVectorCollection & collection)
       // ThresholdEvent
       if (collection[i].getAntecedent().getImplementation()->getId() != rootCauseId)
         throw NotYetImplementedException(HERE) << "Root cause not found";
-      composedEvent_ = composedEvent_.join(collection[i]);
+      //composedEvent_ = composedEvent_.join(collection[i]);
```

Comment:
we still need composedEvent_ for subset

Reply:
The SubsetAlgorithm cannot work if `computedEvent_` attribute is removed.

Reply:
Normally, this issue should be solved now. I have retained the composedEvent_ attribute, but it is only populated when explicitely required either by the user or by an algorithm like SubsetSampling.

As a side-effect, the code is now more simple IMO.

Reply:
@mbaudin47 : Actually, the existing unit test already tests it. This is why it previously failed on CI machines.